build.sc

```diff
@@ -8,45 +8,41 @@ import mill.contrib.buildinfo.BuildInfo
 
 object chisel3 extends mill.Cross[chisel3CrossModule]("2.13.6", "2.12.14")
 
```

Comment:
It would be better to make these build changes separately

Reply:
I should create another PR for mill. I forgot it. ;p